index.ts

```diff
@@ -23,12 +26,39 @@ interface ModalStyles {
   padding?: string;
 }
 
+interface ResponseListenerData {
+  action: string;
+  resolve: (response: any) => void;
+}
+
+const preconnectGoogleFonts = document.createElement('link');
+preconnectGoogleFonts.rel = 'preconnect';
+preconnectGoogleFonts.href = 'https://fonts.googleapis.com';
+document.head.appendChild(preconnectGoogleFonts);
```

Comment:
since lib for both client and server environments make sure to check for document global object, if already covered lmk